docs/release_notes/release_notes-0.15.0.rst

```diff
@@ -7,6 +7,7 @@ Key Features and Updates
   * FIX-#4376: Upgrade pandas to 1.4.2 (#4377)
   * FIX-#3615: Relax some deps in development env (#4365)
   * FIX-#4370: Fix broken docstring links (#4375)
+  * FIX-#4330: Override the memory limit to start ray 1.11.0 on Macs (#4335)
```

Comment:
Please add your GH nickname to `Contributors` list in this file and fix conflicts.

Reply:
Done.